src/client/Microsoft.Identity.Client/WsTrust/MexDocument.cs

```diff
@@ -225,6 +228,26 @@ private void SetPolicyEndpointAddresses(XContainer mexDocument)
             }
         }
 
+        private IEnumerable<XElement> FindElements(XContainer mexDocument, XNamespace xNamespace, string element)
+        {
+            IEnumerable<XElement> bindingElements = mexDocument.Elements().First().Elements(xNamespace + element);
+
+            //Unable to find metadata information in root node. Attempting to search document.
+            if (bindingElements.Count() == 0)
+            {
+                bindingElements = mexDocument.Elements().DescendantsAndSelf().Elements(xNamespace + element);
```

Comment:
Could this be changed to `mexDocument.Elements().DescendantsAndSelf(xNamespace + element);`?

Reply:
I tried this earlier and it did not work